lib/stripe_mock/webhook_fixtures/payment_intent.processing.json

```diff
@@ -0,0 +1,62 @@
+{
+  "id": "evt_00000000000000",
+  "object": "event",
+  "api_version": "2018-02-28",
+  "created": 1606405356,
+  "data": {
+    "object": {
+      "id": "pi_00000000000000",
+      "object": "payment_intent",
+      "allowed_source_types": ["card", "sepa_debit"],
+      "amount": 200,
+      "amount_capturable": 0,
+      "amount_received": 0,
+      "application": null,
+      "application_fee_amount": null,
+      "canceled_at": null,
+      "cancellation_reason": null,
+      "capture_method": "automatic",
+      "charges": null,
+      "client_secret": "pi_00000000000000",
+      "confirmation_method": "automatic",
+      "created": 1606402945,
+      "currency": "eur",
+      "customer": "cus_00000000000000",
+      "description": null,
+      "invoice": null,
+      "last_payment_error": null,
+      "livemode": false,
+      "metadata": {},
+      "next_action": null,
+      "next_source_action": null,
+      "on_behalf_of": null,
+      "payment_method": null,
+      "payment_method_options": {
+        "card": {
+          "installments": null,
+          "request_three_d_secure": "automatic"
+        }
+      },
+      "payment_method_types": ["card", "sepa_debit"],
+      "receipt_email": null,
+      "review": null,
+      "setup_future_usage": null,
+      "shipping": null,
+      "source": null,
+      "statement_descriptor": "ACME Corp",
+      "statement_descriptor_suffix": null,
+      "status": "requires_source",
```

Comment:
Shouldn't this be `processing`?

Reply:
Well spotted, I copied the fixtures over from the other PR and missed out on these